app/utils/Navigator.scala

```diff
@@ -229,10 +229,13 @@ class Navigator @Inject extends Logging with FeatureSwitching {
     ),
     RegisteringBusinessId -> { userAnswers =>
       userAnswers.registeringBusiness match {
-        case Some(true) if userAnswers.businessEntity.contains(NETP) => pageIdToPageLoad(TaxableSuppliesInUkId)
-        case Some(true) if userAnswers.businessEntity.contains(Overseas) => pageIdToPageLoad(TaxableSuppliesInUkId)
-        case Some(true) => pageIdToPageLoad(RegistrationReasonId)
-        case Some(false) => pageIdToPageLoad(VATExceptionKickoutId)
+        case Some(OwnBusiness) if userAnswers.businessEntity.contains(NETP) || userAnswers.businessEntity.contains(Overseas) =>
+          pageIdToPageLoad(TaxableSuppliesInUkId)
+        case Some(OwnBusiness) => pageIdToPageLoad(NinoId)
```

Comment:
should redirect to RegistrationReasonId, same with
case Some(SomeoneElse) if isEnabled(ThirdPartyTransactorFlow)

probably happened because the reg reason story was merged while this was in development